config-templates/vmware/vsphere-csi-driver/2.5.1/custom-parameters.json

```
@@ -0,0 +1,50 @@
+[
+    {
+        "description": "Provide the vCenter Username",
+        "displayname": "vCenter Username",
+        "name": "vcenter-username",
+        "required": "true",
+        "category": "secret",
+        "obfuscate": "true",
+        "mutable": "true"
+    },
+    {
+        "description": "Provide the vCenter Password",
+        "displayname": "vCenter Password",
+        "name": "vcenter-password",
+        "required": "true",
+        "category": "secret",
+        "obfuscate": "true",
+        "mutable": "true"
+    },
+    {
+        "description": "Insecure flag determines if connection to VC can be insecured",
+        "displayname": "insecure flag",
```

```suggestion
        "displayname": "Secure connection",
```
I think this should be the equivalent of saying: "Do you want to use a secure connection?" With the default as `true` - If you don't, this would be false (unchecked)